l5kit/l5kit/rasterization/rasterizer_builder.py

```diff
@@ -5,39 +5,46 @@
 import cv2
 import numpy as np
 
-from ..data import DataManager, load_pose_to_ecef, load_semantic_map
+from ..data import DataManager, load_semantic_map
 from .rasterizer import Rasterizer
 from .sat_box_rasterizer import SatBoxRasterizer
 from .sem_box_rasterizer import SemBoxRasterizer
 
 
-def _load_image_and_metadata(image_key: str, data_manager: DataManager) -> Tuple[np.ndarray, dict]:
-    """Loads image from given key and its meatadata. The metadata file should be a file with the same key except for
-    having a .json extension instead.
+def _load_metadata(meta_key: str, data_manager: DataManager) -> dict:
+    """
+    Load a json metadata file
 
     Args:
-        image_key (str): key to the image (e.g. ``maps/my_satellite_image.png``)
+        meta_key (str): relative key to the metadata
         data_manager (DataManager): DataManager used for requiring files
 
-    Raises:
-        Exception: Image or metadata is missing or invalid
+    Returns:
+        dict: metadata as a dict
+    """
+    image_metadata_path = data_manager.require(meta_key)
```

Comment:
I see `image_metadata_path`, `dataset_meta_key`, `data_meta_key`, `dataset_metadata_filename`, shall we unify the naming? xD

Reply:
that's a typo you're right :) I'll clean the namespace